README.md

```diff
@@ -82,6 +82,17 @@ There is also a demo application:
     cd tensorflow-mnist
     stack --docker --docker-image=$IMAGE_NAME build --exec Main
 
+If you want to use GPU you can do:
+
+    IMAGE_NAME=tensorflow/haskell:1.0.1-gpu
+    docker build -t $IMAGE_NAME docker/gpu
+
+We need stack to use nvidia-docker by using a 'docker' wrapper script.
+
+    ln -s `pwd`/tools/nvidia-docker-wrapper.sh <somewhere in your path>/docker
```

Comment:
I think it is worth including a note that this will shadow the normal `docker` command.

Filed https://github.com/commercialhaskell/stack/issues/3499 to ask for a nicer way to do this.

(Also, interestingly, I stumbled upon a blog in Japanese where someone documented another workaround, but this one seems simpler: https://shigekikarita.github.io/blog/2017/07/08/023.html)

Reply:
I'll add a note. I have heard rumours that nvidia-docker v2 will be integrated into docker proper. I don't have a source for this.